plugins/lsp/language-servers/default.nix

```diff
@@ -502,6 +502,11 @@ let
       serverName = "rnix";
       package = null;
     }
+    {
+      name = "ruby-lsp";
+      description = "ruby-lsp for Ruby";
+      serverName = "ruby_lsp";
```

Comment:
Should this define a package (`pkgs.ruby-lsp`) or is that handled automatically elsewhere?

Reply:
This is done automatically yes: https://github.com/nix-community/nixvim/blob/0ba2ea5416485d3f6552a108710711acd688cfef/plugins/lsp/language-servers/_mk-lsp.nix#L12